core/lib/Thelia/Core/Template/Loop/Category.php

```diff
@@ -46,10 +46,11 @@
  *
  * {@inheritdoc}
  * @method int[] getId()
- * @method int getParent()
- * @method int getExcludeParent()
- * @method int getProduct()
- * @method int getExcludeProduct()
+ * @method int[] getParent()
+ * @method int[] getExcludeParent()
+ * @method int[] getProduct()
```

Comment:
Hello,
This changement breaks the method ```findPk``` Line 168 and 178, please use ```findPks``` and squash your commits ;)